css/style.css

```diff
@@ -1,6 +1,96 @@
 /* General page styles */
 
 /* We should use a mobile first approach having out breack point at 768px */
+/*Comodities */
+body {
+  font-family: "Roboto", sans-serif;
+  margin: 20px;
+}
+
+.carousel-image{
+  width: 100%;
+  height: 100%;
+  max-height: 233px;
+  max-width: 367px;
+}
+.carrouselContainer {
+  width: 95%;
+  max-width: 1000px;
+  margin: 0 auto;
+  box-sizing: border-box;
+}
+.title-gallery {
+  width: 25px bold;
+}
+.line {
+  width: 90px;
+  height: 1px;
+  margin-left: 0;
+  background-color: #63ccff;
+  border: 0;
+}
+.gallery {
+  padding-top: 10%;
+  width: 100%;
+  display: flex;
+  box-sizing: border-box;
+}
+.gallery__bottom {
+  font-size: 50px;
+  width: 16%;
+  height: 24px;
+  font-weight: bold;
+  position: relative;
+}
+.gallery__bottom:hover {
+  cursor: pointer;
+  transform: scale(1.2, 1.2);
+}
+.back {
+  margin-top: 74.1px;
+  left: 17%;
+  color: #fefefe;
+  position: relative;
+}
+.ahead {
+  margin-top: 74.1px;
+  right: 41px;
+  color: #fefefe;
+}
+.pointer {
+  border-radius: 50%;
+  display: flex;
+  padding-left: 42%;
+  margin: 10px;
+  list-style-position: initial;
+  list-style-type: disc;
+}
+.pointer__point {
+  border-radius: 50%;
+  padding-left: 0%;
+  margin-left: 6%;
+}
+.pointer:hover {
+  cursor: pointer;
+  transform: scale(1.2, 1.2);
+}
+.button__gallery {
+  position: absolute;
+  left: 40%;
+  height: 30px;
+  color: #fefefe;
+  margin-top: 5%;
+  margin-bottom: 10%;
+  border: 0;
+  margin-left: auto;
+  margin-right: auto;
+}
```

Comment:
lines `82, 83, 85 and 86` can be unified as `margin: 5% auto 10%;`.

also lets discuss if for now it is ok to use this custom class or something like `button--medium` should be used instead.

Note `button--medium` and other sizes still need an overhaul to scale correctly.

@josemariogutierrez what do you think?

Reply:
fixed lines